lib/Checks.groovy

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file holds several functions used to perform standard checks for the nf-core pipeline template.
+ */
```

Comment:
I am not sure this library file structure follows the official template - see RNAseq 

![image](https://user-images.githubusercontent.com/2357039/152503846-458a835a-e9e0-4f6b-b753-ccc8289058f6.png)

Reply:
im also not sure if it has to or not!